content/200-concepts/100-components/02-prisma-client/260-client-extensions/index.mdx

```diff
@@ -29,6 +29,19 @@ There are four types of extension. You can use each type to customize a differen
 
 To create Prisma Client extensions, use the `$extends` [client-level method](/reference/api-reference/prisma-client-reference#client-methods), and the appropriate extension type (`model`, `client`, `result` or `query`).
 
+For example, the following code uses the `model` extension type to extend the `user` model:
+
+```ts
+const xprisma = prisma.$extends({
+  name: `signUp`,  // (Optional) This extension name appears in error logs and tracing
+  model: {  // This is a model type extension
```

Comment:
"model" could be in quotes here

(why can I not suggest edits here!?)

Reply:
Thanks, I'll action this